model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -992,13 +992,15 @@ message SideInput {
 // An environment for executing UDFs. By default, an SDK container URL, but
 // can also be a process forked by a command, or an externally managed process.
 message Environment {
+  // Deprecated
+  string url = 1;
```

Comment:
AFAIK it has to remain the first field (i.e. assigned to `1`) because it breaks the backwards-compatibility of the proto otherwise. I'm assuming Dataflow code depends on this.

I agree it should be removed as quickly as possible. We should agree on a timeline for this to avoid it sticking around forever. Alternatively, would it be possible to delay merging this PR until the integration is fixed?

Reply:
A vote passed to get the Dataflow worker code out to sit with the Apache Beam codebase. This should help allow us to make these kinds of breaking changes sooner/faster.

Reply:
Yes, field number has to remain the same to keep the backward compatibility.
We can delay this merge till worker coder is migrated to Git.
By when are we planning to get worker code in  Git?

Reply:
Somewhere between 1-4 weeks, depends on how much time people get to work on it and how long Google internal process takes.

Reply:
I don't think we should/need to delay this PR. I was looking for clarification why we do what we do. Please add a comment why we retain this field. Otherwise this is a bit mysterious to folks not working on Dataflow.

Reply:
I agree, no need to delay further. Was just checking whether we could skip the transition phase if it were a matter of days.

Reply:
Shall we get the review then :)